subsys/usb/class/bt_h4.c

```diff
@@ -0,0 +1,314 @@
+/*
+ * Wireless / Bluetooth USB class
+ *
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <sys/byteorder.h>
+
+#include <usb/usb_device.h>
+#include <usb/usb_common.h>
+#include <usb_descriptor.h>
+
+#include <net/buf.h>
+
+#include <bluetooth/buf.h>
+#include <bluetooth/hci_raw.h>
+#include <bluetooth/l2cap.h>
+
+#define LOG_LEVEL CONFIG_USB_DEVICE_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(usb_bt_h4);
+
+static K_FIFO_DEFINE(rx_queue);
+static K_FIFO_DEFINE(tx_queue);
+
+/* ACL data TX buffers */
+#if defined(CONFIG_BT_CTLR_TX_BUFFERS)
+#define BUF_COUNT MAX(CONFIG_BT_HCI_CMD_COUNT, CONFIG_BT_CTLR_TX_BUFFERS)
+#else
+#define BUF_COUNT MAX(CONFIG_BT_HCI_CMD_COUNT, 4)
+#endif
+
+#if defined(CONFIG_BT_CTLR_TX_BUFFER_SIZE)
+#define BT_L2CAP_MTU (CONFIG_BT_CTLR_TX_BUFFER_SIZE - BT_L2CAP_HDR_SIZE)
+#else
+#define BT_L2CAP_MTU 64
+#endif
+
+/* Data size needed for CMD and ACL buffers */
+#define BUF_SIZE MAX(BT_BUF_RX_SIZE, BT_L2CAP_BUF_SIZE(BT_L2CAP_MTU))
+NET_BUF_POOL_DEFINE(tx_pool, BUF_COUNT, BUF_SIZE, sizeof(u8_t), NULL);
+
+#define BT_H4_OUT_EP_ADDR		0x01
+#define BT_H4_IN_EP_ADDR		0x81
+
+#define BT_H4_OUT_EP_IDX		0
+#define BT_H4_IN_EP_IDX			1
```

Comment:
Indentation is off.

Reply:
It juts github that don't seem to format tab properly, will have to use space to indent.

Reply:
I think it should be spaces anyway.